openmls/src/framing/message.rs

```diff
@@ -28,7 +28,7 @@ use crate::error::LibraryError;
 ///     // ... continued in [MlsMessageBody] ...
 /// } MLSMessage;
 /// ```
-#[derive(PartialEq, Debug, Clone)]
+#[derive(PartialEq, Debug, Clone, TlsSerialize, TlsSize, TlsDeserialize)]
 pub(crate) struct MlsMessage {
     pub(crate) body: MlsMessageBody,
```

Comment:
I presume the version is added later?

Reply:
Yes, it's probably a relatively small change, but I didn't want to stretch this PR further. It's tracked in #979 and I'll tackle it in a follow-up.